pallets/parachain-staking/src/lib.rs

```diff
@@ -982,18 +971,7 @@ pub mod pallet {
 				// 50% of the max block weight as safety margin for computation
 				weight.reads(reads) + weight.writes(writes) + 250_000_000_000
 			} else {
-				// already ran the other migration
-				if !<FixTotalCountedBackingExecuted<T>>::get() {
-					let (mut reads, mut writes) =
-						patch_total_counted_total_backing_to_include_self_bond::<T>();
-					reads += 1u64;
-					writes += 1u64;
-					<FixTotalCountedBackingExecuted<T>>::put(true);
-					// 20% of the max block weight as safety margin for computation
-					weight.reads(reads) + weight.writes(writes) + 100_000_000_000
-				} else {
-					weight.reads(1u64)
-				}
+				weight.reads(1u64)
```

Comment:
What's the idea here? I thought you had a storage item to make sure this is idempotent.

Reply:
It was a submigration, I removed it anyway